src/commands/utils/subscriptions.ts

```diff
@@ -57,3 +60,12 @@ export async function getSubscription(
     }
     return { succeeded: true, result: subResult };
 }
+
+export async function handleNoSubscriptionsFound(): Promise<void> {
+    const noSubscriptionsFound = "No subscriptions were found. Set up your account if you have yet to do so.";
```

Comment:
📓 Thought share: --> scenario for latter - if we have any msdn doc we want to relay this to. Let's do this helpful stuff post release 0?

Reply:
found uber level doc for subscription/billing creation https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/

Reply:
Updated doc link